SoftLayer/managers/monitor.py

```diff
@@ -0,0 +1,85 @@
+"""
+    SoftLayer.monitor
+    ~~~~~~~~~~~~~~~~~~
+    Monitoring Manager/helpers
+
+    :license: MIT, see LICENSE for more details.
+"""
+
+
+from SoftLayer import utils
+
+
+class MonitoringManager(utils.IdentifierMixin, object):
```

Comment:
`utils.IdentifierMixin` isn't needed. You don't specify ways to look up whatever a "monitor" would be based on something other than IDs.